src/components/Auth/Login/Login.react.js

```diff
@@ -236,27 +247,34 @@ class Login extends Component {
 				  <div className="loginForm">
 				    <form id='loginform' onSubmit={this.handleSubmit}>
 				      <div>
-				        <input 
-				         type = "text"
-				         name="email"
-				         style={fieldStyle}
-				         value={this.state.email}
-				         onChange={this.handleChange}
-				         errorText={this.emailErrorMessage}
-				         placeholder="Email" />
-
-				        <input
-					 type = "password"
-				         name='password'
-				         style={fieldStyle}
-				         value={this.state.password}
-				         onChange={this.handleChange}
-				         placeholder='Password' />
+
+				      <TextField
+				        name="email"
+				        style={fieldStyle}
+				        className="fieldStyle"
+				      	value={this.state.email}
+					placeholder="Email"
+				      	onChange={this.handleChange}
+				        underlineStyle={{display: 'none'}} />
+
+				      <PasswordField
+				        name="password"
+				        style={fieldStyle}
+				        className="fieldStyle"
+				        value={this.state.password}
+					placeholder="Password"
+				        onChange={this.handleChange}
+				        inputStyle={inputStyle}
+				        underlineStyle={{display: 'none'}}
+				        disableButton={true}
+				        visibilityButtonStyle={{'display':'none'}}
+				        visibilityIconStyle={{'display':'none'}} />
 
 				        <RaisedButton
 				        label="Login"
 				        type="submit"
-					style={{'width':'10%'}}
+					className='loginbtn'
+					style={{'marginLeft':'10px'}}
 				        backgroundColor={
 				        UserPreferencesStore.getTheme() === 'light' ? '#4285F4' : '#4285F4'}
```

Comment:
Can you please check for lines of code like these and rectify them? This isn't related to this PR but is something that should be taken care of. The PR itself looks good to me. Thanks :-)

Reply:
@Akshat-Jain Implementing different input fields require changes for other components as well to keep it responsive. Thanks.